src/audio/drc/testbench_drc_asin/drc_asin_fixed.c

```diff
@@ -0,0 +1,115 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright(c) 2021 Intel Corporation. All rights reserved.
+//
+// Author: Shriram Shastry <malladi.sastry@linux.intel.com>
+/* Include Files */
+#pragma warning (disable : 4996)
+#pragma warning (disable : 4013)
```

Comment:
btw, what are these for ? can we have a comment for each, I'm guessing it's an XCC warning ?

Reply:
(1) .error     C4996: 'fopen': This function or variable may be unsafe
(2). warning C4013: undefined; assuming extern returning int
xtensa compiler

Reply:
ok, but you should fix this in the code and not ignore, btw can you add more context to the commit message.